model/API.inc.php

```diff
@@ -1324,13 +1324,13 @@ public static function getSearchParamValues($params, $param) {
 		return $sets;
 	}
 	
-	public static function getRateLimit($userID, $ip) {
+	public static function getRateLimit($params) {
 		// For individual and authorized users. One user don't need too many requests
-		if ($userID) {
+		if ($params['userID']) {
 			// 100 request burst with 10 request per second rate limit
 			return [
 				'logOnly' => false,
-				'bucket' => $userID . '_' . $ip,
+				'bucket' => $params['userID'] . '_' . $params['IP'],
```

Comment:
`$params['ip']` (abbreviations lowercase at beginning of variable name, capitalized otherwise)